tests/fixtures/trailing_line_empty.csv

```diff
@@ -0,0 +1,3 @@
+a,b
+0,0
+0,1
```

Comment:
It does not look like there's a trailing empty line here, maybe your IDE trimmed it ?

EDIT: Or maybe you meant that the file ends with a `\n` ? Maybe `trailing_newline` would be clearer in that case ?

Reply:
Ahhh I think the pre-commit did that '-_-

Reply:
Actually no, It's github that hides this new line, and it happends my vi configuration does that too :p
But if you `nano` the file (e.g. `wget https://raw.githubusercontent.com/ToucanToco/peakina/9360ed9d72f058dc02f3bc56e6ca08e9695aa19f/tests/fixtures/trailing_line_empty.csv`) you'll see the trailing line

Reply:
Yeah `cat -e` of the file on your branch gives 
``` 
a,b$
0,0$
0,1$
```
So we're good :smile: . With the file's initial name, I thought what you wanted was 
```
a,b$
0,0$
0,1$
$
```
But it's clearer with the new name. Anyway, everything looks fine now :+1: